scanner/src/main/kotlin/LocalScanner.kt

```diff
@@ -320,7 +320,7 @@ abstract class LocalScanner(
 
```

Comment:
Commit message nit: I think this should be "...this could have led to name *clashes* in the...".